org.wezfurlong.wezterm.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.wezfurlong.wezterm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "wezterm",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=home:ro",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.portal.Desktop",
+        "--talk-name=org.freedesktop.portal.Settings",
```

Comment:
Are these two actually needed? I don't think I've seen them before allowed explicitly.

Reply:
I mentioned this briefly above: yes, wezterm uses dbus to access the desktop portal to reason about desktop settings, particularly dark mode.

Reply:
but did you check without them? I believe these are granted by default as it is the portal.

Reply:
will test and report back

Reply:
Verified that they aren't needed: removed!